src/App.scss

```diff
@@ -30,9 +30,13 @@
     padding-right: 2px;
   }
 
+  .fact-header-cell {
+    td {
+      border-bottom: 3px solid #0000FF;
```

Comment:
We want to use `#39a5dc` for this color. Can you add coloring of the caret as well?